### PR TITLE
Added a remark about uap10:TrustLevel

### PR DESCRIPTION
**Pull request by ahmed605**
This remark describes the requirements for uap10:TrustLevel for partialTrust apps

### CODE REVIEW

**Comment by Jak-MS** [comment]
@drewbatgit 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by MeaK199 — APPROVED** [review]

File: winrt-related-src/schemas/appxpackage/uapmanifestschema/element-application.md [hunk 1]
```diff
@@ -109,6 +109,8 @@ So these combinations are valid, and they all produce identical results:
 
 Similarly, if `uap10:TrustLevel="appContainer"`, then on those older systems `EntryPoint` must be set to "windows.partialTrustApplication`.
 
+Setting `uap10:TrustLevel="mediumIL"` while `uap10:RuntimeBehavior="windowsApp"` (default behavior for UWP apps) or `EntryPoint="windows.partialTrustApplication"` requires the `Microsoft.coreAppActivation_8wekyb3d8bbwe` Custom Capability, you can read more about this capability here in [Custom Capabilities](/windows/uwp/packaging/app-capability-declarations#custom-capabilities)
```
Comment (DrusTheAxe) [inline winrt-related-src/schemas/appxpackage/uapmanifestschema/element-application.md]: Long line. Please keep markdown file lines to 100chars

Comment (DrusTheAxe) [inline winrt-related-src/schemas/appxpackage/uapmanifestschema/element-application.md]: >Setting `uap10:TrustLevel="mediumIL"` while `uap10:RuntimeBehavior="windowsApp"` (default behavior for UWP apps) or `EntryPoint="windows.partialTrustApplication"`

That's an error as TrustLevel contradicts EntryPoint. If both TL+EP (or RB+EP) are specified they must have consistent values, else error.

SUGGESTION: `Setting `uap10:TrustLevel="mediumIL"` while `uap10:RuntimeBehavior="windowsApp"` requires... This is also true if `uap10:TrustLevel="mediumIL"` and `EntryPoint` is any value other than `"windows.fullTrustApplication"` or `"windows.partialTrustApplication"`.

**Review by DrusTheAxe — APPROVED** [review]
Critical correction to the comment otherwise good

Universal app + MediumIL requires the custom cap. Only way to define that permutation is TL=mediumIL + RB=windowsApp --OR-- TL=mediumIL and EP!=windows.fullTrustApplication and EP!=windows.partialTrustApplication. If EP is any other value it means you're declaring a Universal app, same as if you'd said so explicitly via RB=windowsApp

File: winrt-related-src/schemas/appxpackage/uapmanifestschema/element-application.md (hunk 1 above)
Comment (ahmed605) [reply winrt-related-src/schemas/appxpackage/uapmanifestschema/element-application.md]: I see, fixed

Comment (ahmed605) [reply winrt-related-src/schemas/appxpackage/uapmanifestschema/element-application.md]: Fixed

**Review by drewbatgit — APPROVED** [review]

**Pull request merged**